src/intro.js

```diff
@@ -12,30 +12,38 @@
  * Date: @DATE
  */
 
-(function( global, factory ) {
+(function() {
+	/* jshint -W117 */
+	var globalSetTimeout = setTimeout,
```

Comment:
Of course we'd be able to reduce the size significantly by avoiding this additional scope to save the globals but then we have to rename `setTimeout` & friends in the code.